index.ts

```diff
@@ -1,6 +1,7 @@
-export { getBodyEnd } from './slots';
+export { getBodyEnd } from "./slots";
```

Comment:
Oh - I think the prettier/eslint config I added the other day _should_ be enforcing single quotes (as we do elsewhere). If you rebase on `master` does that make a difference?